src/Product/ProductInfo.cs

```diff
@@ -8,15 +8,15 @@ namespace Azure.DataApiBuilder.Product;
 
 public static class ProductInfo
 {
-    public const string DEFAULT_VERSION = "1.0.0";
+    public const string DEFAULT_VERSION = "0.0.0";
```

Comment:
I believe default was given 1.0.0 for tests, because it is what gets generated automatically by assembly when nuget is not installed, or installed without specifying a version.

Reply:
I see. The point of using something different than 1.0.0 is to make it possible to understand when 1.0.0 really means 1.0.0 vs when it means there was an issue getting the version number

Reply:
I updated the unit tests and, at least locally, it seems everything is fine with "0.0.0"